fontforge/ffpython.h

```diff
@@ -109,7 +109,7 @@ extern void PyFF_Glyph_Set_Layer(SplineChar *sc,int layer);
 typedef struct ff_point {
     PyObject_HEAD
     /* Type-specific fields go here. */
-    float x,y;
+    double x,y;
```

Comment:
float was causing conversion problems. I don't believe there is a dependency between this representation and the one chosen for "internal" splines.